.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       fail-fast: false
       matrix:
         gradle-task: [test, ktlintCheck]
-        kotlin-version: [1.3.72, 1.4.0-rc, 1.4.20-dev-2803]
```

Comment:
Just curious, you don't want to continue testing on `1.3.72` to ensure keep backwards compatibility for a bit?